Main/ClientApp/src/Actions/CollectionActions.ts

```diff
@@ -49,6 +49,40 @@ export async function loadSharedCollections() {
   });
 }
 
+export async function loadContributorsSharedCollections() {
+  let contributorsSharedCollections = await SharedCollectionsApi.getContributorsSharedCollections();
+  Dispatcher.dispatch({
+    actionType: ActionTypes.LOAD_CONTRIBUTORS_SHARED_COLLECTIONS,
+    payload: { contributorsSharedCollections: contributorsSharedCollections },
+  });
+}
+
 export async function shareCollection(shareData: SharedCollectionData) {
-  await SharedCollectionsApi.shareCollection(shareData);
+  let success = await SharedCollectionsApi.shareCollection(shareData);
+  if (success) {
+    loadCollections();
+    loadSharedCollections();
+    loadContributorsSharedCollections();
+  }
+}
+
+export async function updateSharedCollection(updateData: SharedCollectionData) {
+  let success = await SharedCollectionsApi.updateSharedCollection(updateData);
```

Comment:
If I understand correctly, only thing changing here is user rights. The method name is not representative, the collection is not updated

Reply:
if we are changing something we updating. am i wrong?

Reply:
but name can be changed to be more redeable